org.bluej.BlueJ.json

```diff
@@ -1,22 +1,19 @@
 {
   "app-id" : "org.bluej.BlueJ",
   "branch" : "stable",
-  "runtime" : "org.gnome.Platform",
-  "runtime-version" : "40",
-  "sdk" : "org.gnome.Sdk",
-  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk11"],
   "command": "org.bluej.BlueJ",
   "finish-args" : [
     "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
     "--share=network",
     "--socket=x11",
     "--share=ipc",
-    "--socket=wayland",
```

Comment:
Why'd this sneak in?

Reply:
It is a mistake made by me. I will revert it. But does using socket=x11 and socket=wayland work together? Which will be picked? Should it not be socket=fallback-x11 and socket=wayland?

Reply:
They can be used together yes, it results in both being available to the app (which given the nature of BlueJ, is probably the most appropriate at this stage)

Reply:
`fallback-x11` mean that if there is a wayland session there is no X11 socket available. Otherwise BOTH are available if both specified.

Reply:
The application doesn't support wayland and you run a wayland session, it is expect that it will fail with the `fallback` and that mean you should only have the `x11` socket. In general both `wayland` and `x11` is a mistake, but there are exceptions.